### DISCUSSION TITLE
Where are the videos on Helix?

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [helix-editor/helix#1006] [Ideas]
Judging from the current state of the Helix project a lot of people must be using it as their daily editor ( +4K stars , 52 pull request etc ).
So my point is why is the internet dry of any Helix videos comparing it, showing its features, its weakness, discussing about it or just generally using it...
I'm not a Helix user myself but I'm really curious as I watch this project grow and grow into this really powerful and fast editor driven by this active community.
So please at least a couple of you must be willing to make a couple of videos just showing it, showing how you use it what you love and what you hate about it! 
I think it will really help with newcomers especially but generally for the exposure of the project.

**Comment by txtyash** [comment]
Yes, the document tries to assume that we're already familiar with vim so we'd hit n trial to check what some of the key bindings are. It then also expects us to take a look at kakoune's docs so that we try those key-bindings as well to see which few of them also work in helix. Kinda' weird.

Good documentation or something similar to vim-tutor would be great.

**Comment by x86dev** [comment]
+1 here ... I barely use(d) vim, but I'm curious about Helix and would like to have a deeper look at it.

**Comment by kirawi** [comment]
There was this video recently: https://www.youtube.com/watch?v=9Zj-wiQ9_Xw

**Reply by David-Kunz** [reply]
Just wanted to post this :)

**Comment by kirawi** [comment]
There was another video made: https://www.youtube.com/watch?v=__U4A2Q9fNI

**Reply by archseer** [reply]
@amirrezaask thank you! 🎉 It's cool to see you coding in Helix too https://www.youtube.com/watch?v=jQXXZw7p6Yo

**Reply by amirrezaask** [reply]
Helix seems really amazing, I love it for just how easy it is to set it up. there are some configuration I miss but overall i prefer it to neovim

**Reply by amirrezaask** [reply]
love to do more videos on it, and if you could give some hints on what to emphasize is great

**Reply by archseer** [reply]
@amirrezaask Sorry for the late reply, we gave your video a shoutout here https://helix-editor.com/news/release-22-05-highlights/ !

I think covering the basics is good, but even just using Helix is enough, that way we're able to link to an example of someone using the editor :)

**Comment by joegm** [comment]
There was this stream: https://www.youtube.com/watch?v=tNcyWw27dYc

**Comment by jbwyatt4** [comment]
Hello, I am new to Helix. Are there any videos on using Helix with C and ctags (as well as customizing the formatting-I work with the Linux kernel)?
